pyface/qt/QtWebKit.py

```diff
@@ -4,16 +4,18 @@
     from PyQt4.QtWebKit import *
 
 elif qt_api == 'pyqt5':
-    #from PyQt5.QtWebKit import *
-    #from PyQt5.QtWebKitWidgets import *
     from PyQt5.QtWidgets import *
-    from PyQt5.QtWebEngine import *
-    from PyQt5.QtWebEngineWidgets import (
-        QWebEngineHistory as QWebHistory,
-        QWebEngineHistoryItem as QWebHistoryItem,
-        QWebEnginePage as QWebPage,
-        QWebEngineView as QWebView,
-    )
+    try:
+        from PyQt5.QtWebEngine import *
+        from PyQt5.QtWebEngineWidgets import (
+            QWebEngineHistory as QWebHistory,
+            QWebEngineHistoryItem as QWebHistoryItem,
+            QWebEnginePage as QWebPage,
+            QWebEngineView as QWebView,
```

Comment:
question : is `QWebEngineSettings` not available via PyQt5 because we import it from PySide2 but not here.